app/views/scholars_archive/handles/handle_file_404.html.erb

```diff
@@ -0,0 +1,2 @@
+<p> The work you are trying to look at <b><%= @handle_uri %></b> doesn't seem to have the file you requested <b><%= @file  %></b>. If you believe this is an error please notify an administrator. Thank you.</p>
```

Comment:
I'd add a note that this error was logged. Then something like "If you believe this is an error, you can use the Contact Form to let us know."